### PR TITLE
Some fix and features

### LINKED ISSUE #78: Keep the last page view
**Is your feature request related to a problem? Please describe.**
When i open the extension / web app i need to go each time on gifs tab

**Describe the solution you'd like**
Keep the last page view in localstorage and load it on start 👍

### PR DESCRIPTION
**Pull request by T3kstiil3**
GitHub issue/pull request detail

Fixes #78

## Description

- [x] add pen if no theme set ✏️
- [x] use teamsService in home page 🧼
- [x] display day date in schedule page 📅
- [x] keep last view page and redirect on first open 🔗
- [x] restrict theme length 🚨

## Screenshot
 
image

### CODE REVIEW

File: front/src/views/Home.vue [hunk 1]
```diff
@@ -2,31 +2,27 @@
     <div class="home">
         <v-container>
             <v-row dense>
-                <v-col cols="12">
+                <v-col cols="12" v-if="user">
```
Comment (ytvnr) [inline front/src/views/Home.vue]: maybe put the `v-if` on `v-container` ?
If no user we will display a totally different page
wdyt ?

File: front/src/views/Home.vue [hunk 2]
```diff
@@ -74,36 +66,36 @@ export default {
             team: null,
             theme: null,
             gif: null,
-            giphyService: new GiphyService()
+            giphyService: new GiphyService(),
+            teamsService: null,
+            assignedTeam2: null
```
Comment (ytvnr) [inline front/src/views/Home.vue]: we will have to find a better naming on that

File: front/src/views/Schedule.vue [hunk 3]
```diff
@@ -47,7 +47,7 @@
                             persistent
                             @save="save(props.item)"
                         >
-                            <div>{{ props.item.theme }}</div>
+                            <div>{{ props.item.theme || '✏️' }}</div>
```
Comment (ytvnr) [inline front/src/views/Schedule.vue]: maybe we should have always a pen to show it's editable ?

File: front/src/views/Home.vue (hunk 2 above)
Comment (T3kstiil3) [reply front/src/views/Home.vue]: assignedTeamLocal ?

File: front/src/views/Schedule.vue (hunk 3 above)
Comment (T3kstiil3) [reply front/src/views/Schedule.vue]: Can we test a first version like that ?

File: front/src/router/index.js [hunk 4]
```diff
@@ -10,69 +10,86 @@ import firebase from '@/firebase/init';
...
-  } else {
-    next();
-  }
+    const lastRouteName = localStorage.getItem('lastToPath');
+
+    const shouldRedirect = Boolean(
+        to.name === 'home' && lastRouteName && isFirstTransition
+    );
+
+    if (shouldRedirect) next({ name: lastRouteName });
```
Comment (ytvnr) [inline front/src/router/index.js]: i prefer to have brackets even for one line :)

Comment (ytvnr) [reply front/src/router/index.js]: does next prevent the end of the method execution or do we have to force exit with a `return` ?

File: front/src/views/Home.vue (hunk 1 above)
Comment (ytvnr) [reply front/src/views/Home.vue]: ??

File: front/src/views/Home.vue [hunk 5]
```diff
@@ -74,36 +66,36 @@ export default {
...
-                .where('date', '>=', start)
-                .where('date', '<=', end)
-                .get()
+			
+            this.teamsService.getThemesBetweenDates(teamId, start, end)
                 .then(snapshots => {
-                    if (snapshots.size !== 1) {
-                        throw new Error('Cannot have 0 or multiple result');
+                    if(snapshots.size === 0) {
+                        console.info('No theme set today :/');
```
Comment (ytvnr) [inline front/src/views/Home.vue]: 👍

File: front/src/views/Home.vue [hunk 6]
```diff
@@ -116,23 +108,22 @@ export default {
                 });
         }
     },
+    watch: {
+        assignedTeamLocal(id){
```
Comment (ytvnr) [inline front/src/views/Home.vue]:
```suggestion
        storedAssignedTeamId(id){
```

File: front/src/views/Home.vue [hunk 7]
```diff
@@ -116,23 +108,22 @@ export default {
...
+                this.getTeamById(id);
+                this.getThemeByTeamId(id);
+            }
+        }
+    },
     computed: {
         user() {
             return this.$store.getters.user;
         },
         assignedTeam() {
```
Comment (ytvnr) [inline front/src/views/Home.vue]:
```suggestion
        assignedTeamId() {
```

File: front/src/views/Home.vue (hunk 2 above)
Comment (ytvnr) [reply front/src/views/Home.vue]: I did another proposition in comments :)

File: front/src/router/index.js (hunk 4 above)
Comment (T3kstiil3) [reply front/src/router/index.js]: Next force the end of the method. 🙂

**Pull request merged**